modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -318,6 +325,23 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo OutlineStyleInfo = {
+        "OutlineStyle",
+        "Outline Style",
+        "This setting decides the shape of points that have an outline (round, square, "
+        "or lust a line at the bottom). Note that anything but \"Round\" will lead to "
```

Comment:
```suggestion
        "or just a line at the bottom). Note that anything but \"Round\" will lead to "
```

I'd also turn around the "Round" sentence.  I would expect that by default the images are rendered as squares, so instead:

"Note that selecting \"Round\" will lead to the points being drawn as ellipses"

or something like that

Reply:
This gets complicated as the border is not only applied to textured points, but the regular ones as well (which was een supported before this PR).  In that case, I would expect the default value to be "Round" 

And in my opinion, since we are drawing what we call "points", it kind of makes sense to me that the default value always is round.   But if we want to change the default value for when textures are used to be square, that would also be ok

Reply:
Need help deciding what to do here